test/integration/test_create_dss_docker_image.py

```diff
@@ -167,3 +261,71 @@ def test_docker_socket_on_host_touched(dss_container_context, fake_docker_socket
         wait_for_socket_access(container)
 
     assert stat_before == socket.stat()
+
+
+# def test_write_socket_known_gid(socket_inspector, group_changer):
+#     with socket_inspector as inspector:
+#         gid = inspector.get_gid("ubuntu")
+#     group_changer.chgrp(gid, socket_inspector.socket_on_host)
+#     with socket_inspector as inspector:
+#         inspector.assert_jupyter_member_of("ubuntu")
+#         inspector.assert_write_to_socket()
+
+
+@contextmanager
+def added_group(request, base_image, gid, group_name):
+    # dss_docker_image.image_name
```

Comment:
replace with explanation why context instead of fixture, because parameterized with gid